doc/source/whatsnew/v1.1.0.rst

```diff
@@ -352,6 +352,7 @@ Other
   instead of ``TypeError: Can only append a Series if ignore_index=True or if the Series has a name`` (:issue:`30871`)
 - Set operations on an object-dtype :class:`Index` now always return object-dtype results (:issue:`31401`)
 - Bug in :meth:`AbstractHolidayCalendar.holidays` when no rules were defined (:issue:`31415`)
+- Using an `orient` argument not listed inside :meth:`DataFrame.to_dict` will raise a ``ValueError`` (:issue:`32515`)
```

Comment:
@MomIsBestFriend I cannot see why the doc CI are failing with this added line. Not sure if this what's new info had to be added in some other place or I am not supposed  to add this myself :thinking:  Any ideas?

Reply:
@elmonsomiat I am not sure why the docs are failing, maybe try to merge master?

Reply:
it looks like there are too many lines in this section (I tried adding my comment on another section within the same file and it works). Adding this new line breaks it. Not sure if it's OK to leave it as it is, without chaning the whatsnew file!